scss/bitstyles/settings/_typography.scss

```diff
@@ -1,8 +1,16 @@
+@use "sass:list"; // stylelint-disable at-rule-no-unknown
+
 /* stylelint-disable value-keyword-case */
-$bitstyles-font-family-header: "Nunito", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, helvetica, arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol" !default;
-$bitstyles-font-family-body: "Nunito", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, helvetica, arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol" !default;
+$bitstyles-font-family-webfont: 'Nunito' !default;
+$bitstyles-webfont-font-url: '../test/assets/fonts/nunito-v16-latin-regular' !default;
```

Comment:
Why is the font placed in the `test` directory?

Reply:
The content for the styleguide has always lived inside the test directory, because it was testing-related (visual regression tests) not css-related. Now that we’re using storybook it still lives there, with the images we use for the examples in storybook) but if there’s a better place for it I’m totally happy to move it all 🤔